lib/workers/repository/finalise/index.ts

```diff
@@ -15,4 +17,17 @@ export async function finaliseRepo(
     `Action Required: Fix Renovate Configuration`
   );
   await clearRenovateRefs();
+  const prList = await platform.getPrList();
+  if (
+    prList?.some(
+      (pr) =>
+        pr.state === 'merged' &&
+        (pr.title !== 'Configure Renovate' ||
+          pr.title !== config.onboardingPrTitle)
```

Comment:
This will always return true if config.onbaordingPrTitle is set

Reply:
config.onboardingPrTitle should not contain 'Configure Renovate'?
and if pr.title contains dependency update pr the condition will return true 
but if pr.title contains onboarding pr the condition will return false

Reply:
If onboardingPrTitle is "Foo" then this statement will always return true and therefore the onboarding PR will be counted

Reply:
Do you mean that we should remove the second part of the condition?

Reply:
No

Reply:
```
if (
    prList?.some(
      (pr) => pr.state === 'merged' && pr.title !== 'Configure Renovate'
    )
  ) 
```
this condition is not enough to filter onboarding PR?